scripts/justmat/foulplay.lua

```diff
@@ -94,7 +94,7 @@
 -- release the copy button.
 --
 
-require 'er'
+er = require 'er'
```

Comment:
though the global cleanup means it shouldn't matter but maybe make this local anyway?